pkg/cmd/cli/cmd/process.go

```diff
@@ -140,13 +139,30 @@ func RunProcess(f *clientcmd.Factory, out io.Writer, cmd *cobra.Command, args []
 			return err
 		}
 
-		var ok bool
-		templateObj, ok = obj.(*api.Template)
-		if !ok {
-			return fmt.Errorf("cannot convert input to Template: ", reflect.TypeOf(obj))
+		// FIXME: For now, only the first Template item is processed.
+		// When the input is provided in STDIN then all Template objects are loaded
+		// into api.List items, which assumes that we will process them
+		// sequentially. However, this is not (yet) supported as we will have to
+		// deal with multiple parameters set by different templates.
```

Comment:
do you mean multiple override parameters?  ignoring override parameters, we ought to be able to just iterate the list right?